Userland/Libraries/LibJS/Heap/HeapBlock.h

```diff
@@ -27,13 +32,17 @@ class HeapBlock {
 
     ALWAYS_INLINE Cell* allocate()
     {
+        Cell* allocated_cell = nullptr;
         if (m_freelist) {
             VERIFY(is_valid_cell_pointer(m_freelist));
-            return exchange(m_freelist, m_freelist->next);
+            allocated_cell = exchange(m_freelist, m_freelist->next);
+        } else if (has_lazy_freelist())
+            allocated_cell = cell(m_next_lazy_freelist_index++);
```

Comment:
Please don't omit curly braces if any previous or following if/else branch needs them.

Reply:
Ah fair. I wonder if there's a way to get the clang-format config to enforce this rule?